gossipd/routing.c

```diff
@@ -1051,15 +1058,17 @@ bool routing_add_channel_update(struct routing_state *rstate,
 	u64 htlc_minimum_msat;
 	u32 fee_base_msat;
 	u32 fee_proportional_millionths;
+	u64 htlc_maximum_msat;
 	struct bitcoin_blkid chain_hash;
 	struct chan *chan;
 	u8 direction;
 
-	if (!fromwire_channel_update(update, &signature, &chain_hash,
+	if (!fromwire_channel_update_option_channel_htlc_max(update, &signature, &chain_hash,
```

Comment:
How does this work?  Won't we fail to parse old-style channel_updates now?  (Hmm, maybe this is fixed in another commit, since I review one commit at a time...)